mkdocs.yml

```diff
@@ -350,6 +350,16 @@ nav:
           - 'FAQ': variants/pc_engines/faq.md
           - 'Announcement': variants/pc_engines/post-eol-fw-announcement.md
           - 'Firmware write protection': variants/pc_engines/bios-lock.md
+      - 'Hardkernel':
+          - 'Overview': unified/hardkernel/overview.md
+          - 'Building manual': unified/hardkernel/building-manual.md
+          - 'Initial deployment': unified/hardkernel/initial-deployment.md
+          - 'Firmware update': unified/hardkernel/firmware-update.md
+          - 'Recovery': unified/hardkernel/recovery.md
+          - 'Test matrix': unified/hardkernel/test-matrix.md
+          - 'Hardkernel ODROID H4+':
+              - 'Releases': variants/hardkernel_odroid_h4_plus/releases.md
+              - 'Hardware Configuration Matrix': variants/hardkernel_odroid_h4_plus/hardware-matrix.md
```

Comment:
There is no need to make the documentation unified yet. We only have one variant - ODROID H4.

Even though ODROID H4 comes with 3 variants, it is expected to run the same binary.

Reply:
Moved everything to the variants dir. Updated the mkdocs.yml 
https://github.com/Dasharo/docs/pull/903/commits/07655bc698e7aee93f2dd14e44b96ea6d1bee70f